patches/chromium/printing.patch

```diff
@@ -381,20 +378,29 @@ index 9fbea6d0a2dbe55b1d600fbc217dee5aa8ae8cd5..de9bd267e408c02fd4da7d903523c0e6
    // content::BrowserMessageFilter:
    bool OnMessageReceived(const IPC::Message& message) override;
 diff --git a/components/printing/common/print.mojom b/components/printing/common/print.mojom
-index d5ff612320fb3f324b58c91b0c05e0eac689beef..9fafde963ecce596c9514384d2aa11a7540e0d87 100644
+index d5ff612320fb3f324b58c91b0c05e0eac689beef..254810ca174c456b6bf6966954e4e188c2460f15 100644
 --- a/components/printing/common/print.mojom
 +++ b/components/printing/common/print.mojom
-@@ -16,7 +16,7 @@ interface PrintRenderer {
+@@ -4,6 +4,8 @@
+ 
+ module printing.mojom;
+ 
++import "mojo/public/mojom/base/values.mojom";
++
+ // Interface implemented by a class that desires to render print documents for
+ // Chrome print preview.
+ interface PrintRenderer {
+@@ -16,7 +18,7 @@ interface PrintRenderer {
  interface PrintRenderFrame {
    // Tells the RenderFrame to switch the CSS to print media type, render every
    // requested page, and then switch back the CSS to display media type.
 -  PrintRequestedPages();
-+  PrintRequestedPages(bool silent);
++  PrintRequestedPages(bool silent, mojo_base.mojom.DictionaryValue settings);
```

Comment:
what's even in these settings? Would it make sense to use a proper Mojo struct here rather than an untyped `DictionaryValue`?

Reply:
I would say we do unfortunately need to leave this untyped - we could define a whole new struct outlining the settings integral to printing but i think that might introduce more arguably unncessary overhead. 

Since they're all interpreted as an untyped DictionaryValue in the end, this for now feels like a more tenable approach.